### ISSUE TITLE
netx: Invalid argument: net.sourceforge.jnlp.util.optionparser.InvalidArgumentException: [ , /tmp/kvm_<hostname>.jnlp]

### ISSUE THREAD

**Issue by radek-senfeld** [rockyluke/drac-kvm#7]
Hi,
when getJavawsArgs() returns an empty string, javaws refuses to launch: "netx: Invalid argument: net.sourceforge.jnlp.util.optionparser.InvalidArgumentException: [ , /tmp/kvm_<hostname>.jnlp]"

Also there's no -wait flag in my javaws despite OpenJDK 1.8.0 is properly installed (see attached logs). This could be my fault I guess. Maybe the version and thus flag existence should be matched against javaws?

This patch fixed the problem (stdout & stderr redirected for better debugging):

```
diff --git a/main.go b/main.go
index 195c530..4c81f7f 100644
--- a/main.go
+++ b/main.go
@@ -54,8 +54,6 @@ func getJavawsArgs(waitFlag bool) string {
 		strings.Contains(string(slurp[:]), "1.8") {
 		if waitFlag {
 			javawsArgs = "-wait"
-		} else {
-			javawsArgs = ""
 		}
 
 	}
@@ -196,7 +194,10 @@ func main() {
 
 	// Launch it!
 	log.Printf("Launching KVM session with %s", filename)
-	if err := exec.Command(*_javaws, getJavawsArgs(*_wait), filename, "-nosecurity", "-noupdate", "-Xnofork").Run(); err != nil {
+	cmd := exec.Command(*_javaws, getJavawsArgs(*_wait), filename, "-nosecurity", "-noupdate", "-Xnofork")
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	if err := cmd.Run(); err != nil {
 		os.Remove(filename)
 		log.Fatalf("Unable to launch DRAC (%s), from file %s", err, filename)
 	}
```

My environment:

```
rush@jarvis:~$ java -version
openjdk version "1.8.0_181"
OpenJDK Runtime Environment (build 1.8.0_181-8u181-b13-0ubuntu0.16.04.1-b13)
OpenJDK 64-Bit Server VM (build 25.181-b13, mixed mode)

rush@jarvis:~$ javaws -version
icedtea-web 1.6.2 (1.6.2-3ubuntu1)

rush@jarvis:~$ javaws

SYNOPSIS
    javaws [-run-options] jnlp file
    javaws [-control-options]

OPTIONS
    When specifying options, the name of the jnlp file can be after the command, the -jnlp option, an option with no arguments, or after an argument with an option that takes one argument. A html file that launches a jnlp can be specified after the -html option.
    The jnlp-file can either be a url or a local path.
    The JNLP file should only be specified once, whether as a main argument, after -jnlp or through an html file.
    Control options:
    -about               - Shows a sample application.(No argument expected)
    -help                - Prints out information about supported command and basic usage.(No argument expected)
    -license             - Display the GPL license and exit.(No argument expected)
    -viewer              - Shows the trusted certificate viewer.(No argument expected)
    -Xclearcache         - Clean the JNLP application cache.(No argument expected)
    Run options:
    -allowredirect       - Follows HTTP redirects.(No argument expected)
    -arg arg             - Adds an application argument before launching.(Expected one or more arguments)
    -headless            - Disables download window, other UIs.(No argument expected)
    -html                - Location of HTML file to launch (url or file). You can use parameter ALL  or numbers (like 1 2 5) to select applets on page. However experimental, this switch should keep you still in safety.(Expected one or more arguments)
    -jnlp                - Location of JNLP file to launch (url or file).(Exactly one argument expected)
    -nosecurity          - Disables the secure runtime environment.(No argument expected)
    -noupdate            - Disables checking for updates.(No argument expected)
    -param name=value    - Adds an applet parameter before launching.(Expected one or more arguments)
    -property name=value - Sets a system property before launching.(Expected one or more arguments)
    -strict              - Enables strict checking of JNLP file format.(No argument expected)
    -update seconds      - Check for updates.(Exactly one argument expected)
    -verbose             - Enable verbose output.(No argument expected)
    -version             - Print the IcedTea-Web version and exit.(No argument expected)
    -Xignoreheaders      - Skip jar header verification.(No argument expected)
    -xml                 - Uses a strict XML parser to parse the JNLP file.(No argument expected)
    -Xnofork             - Do not create another JVM.(No argument expected)
    -Xoffline            - Prevent ITW network connection. Only cache will be used. Application can still connect.(No argument expected)
    -Xtrustnone          - Instead of asking user, will foretold all answers as no.(No argument expected)
```

**Comment by rockyluke** [comment]
Thanks for your report. I'll check this afternoon.

**Comment by rockyluke** [comment]
I use you diff to fix it in v1.1.2. Thanks again.
